Wire-iOS/Sources/UserInterface/Calling/CallViewController/CallGridConfiguration.swift

```diff
@@ -76,18 +76,25 @@ extension VoiceChannel {
 
     func arrangeStreams(for selfStream: Stream?, participantsStreams: [Stream]) -> StreamArrangment {
         let streamsExcludingSelf = participantsStreams.filter { $0.streamId != selfStreamId }
-
+        let sortedStreamsList = sortByVideo(streamData: streamsExcludingSelf)
         guard let selfStream = selfStream else {
-            return (nil, streamsExcludingSelf)
+            return (nil, sortedStreamsList)
         }
-
-        if callHasTwoParticipants && streamsExcludingSelf.count == 1 {
-            return (selfStream, streamsExcludingSelf)
+        if callHasTwoParticipants && sortedStreamsList.count == 1 {
+            return (selfStream, sortedStreamsList)
         } else {
-            return (nil, [selfStream] + streamsExcludingSelf)
+            return (nil, [selfStream] + sortedStreamsList)
         }
     }
 
+    func sortByVideo(streamData: [Stream]) -> [Stream] {
+        let sortedData = streamData.sorted {
+            guard let videoStatusArgument0 = $0.videoState?.isSending else { return false }
+            guard let videoStatusArgument1 = $1.videoState?.isSending else { return false }
+            return videoStatusArgument0 && !videoStatusArgument1
+        }
+        return sortedData
+    }
```

Comment:
`sortedData` isn't strictly necessary as it is directly returned after being set.

```suggestion
    func sortByVideo(streamData: [Stream]) -> [Stream] {
        return streamData.sorted {
            guard let videoStatusArgument0 = $0.videoState?.isSending else { return false }
            guard let videoStatusArgument1 = $1.videoState?.isSending else { return false }
            return videoStatusArgument0 && !videoStatusArgument1
        }
    }

```